server/app/vectors.py

```diff
@@ -80,3 +116,106 @@ def add_vector(
     raise HTTPException(
         status_code=status.HTTP_400_BAD_REQUEST, detail="Invalid vector"
     )
+
+
+@router.post("/leveln/", response_model=schemas.VectorOut)
```

Comment:
This can definitely be merged with the `POST /vectors/` endpoint.

Reply:
As discussed, `POST /vectors/` and `POST /leveln/` are bad names. Change to `POST /submit/genbank/` and `POST /submit/vector/`

Reply:
Applied is latest commit.